.circleci/config.yml

```diff
@@ -54,6 +54,8 @@ jobs:
       - checkout
       - run:
           command: |
+            sudo curl -L "https://github.com/docker/compose/releases/download/1.27.4/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+            sudo chmod +x /usr/local/bin/docker-compose
```

Comment:
So this was previously using a `docker-compose` present in the base `ubuntu-1604` image? Ubuntu 16 is past EOL, so we should probably use a newer ubuntu image anyway. Can you check if using the latest LTS ubuntu image gives us a working docker-compose out of the box?

Reply:
I can (will do on Monday), I didn't want to just upgrade the base image as I wasn't sure if it's used anywhere else in the project and would want to avoid different environment in prod and testing.